config/grafana/status.json

```diff
@@ -88,6 +88,86 @@
       "title": "Enabled Modules",
       "type": "stat"
     },
+    {
+      "datasource": null,
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "custom": {
+            "align": null,
+            "filterable": false
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 7,
+        "w": 24,
+        "x": 0,
+        "y": 11
+      },
+      "id": 12,
+      "options": {
+        "showHeader": true
+      },
+      "pluginVersion": "7.5.34",
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "lifecycle_mgr_module_state{state=\"Error\"} > 0",
```

Comment:
I tested this on DEV but it is not showing any entries although there are modules in Error there

Reply:
As discussed, the issue is that we no longer update the module status if the remote kyma is unreachable. Another issue will be created to address this